packages/components/src/components/SquigglePlayground/LeftPlaygroundPanel/index.tsx

```diff
@@ -102,7 +123,36 @@ export const LeftPlaygroundPanel = forwardRef<LeftPlaygroundPanelHandle, Props>(
           icon={Bars3CenterLeftIcon}
           onClick={editorRef.current?.format}
         />
-        <SetttingsMenuItem onClick={() => openModal("settings")} />
+        <Dropdown
+          render={() => (
+            <DropdownMenu>
+              <DropdownMenuActionItem
+                title="Configuration"
+                icon={AdjustmentsVerticalIcon}
+                onClick={() => openModal("settings")}
+              />
+
+              {
+                // experimental, won't always work, so disabled for now
+                /* <DropdownMenuActionItem
+                title="Find in Viewer"
+                icon={AdjustmentsVerticalIcon}
+                onClick={() => editorRef.current?.viewCurrentPosition()}
+              /> */
+              }
+              <DropdownMenuActionItem
+                title="Dependency Graph"
+                icon={PuzzleIcon}
```

Comment:
I'd have expected some graph would have been better, but this is fine

Reply:
I couldn't find anything in heroicons. Also a most popular icon that looks like a graph is a share icon (https://en.wikipedia.org/wiki/Share_icon#/media/File:Sharethis.svg), so it's tricky.

Reply:
Makes sense. This seems fine for now.